src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-rtree.py

```diff
@@ -11,10 +11,8 @@
 # ------------------------------------------------------------------
 
 from PyInstaller.compat import is_pure_conda
+from PyInstaller.utils.hooks import collect_dynamic_libs, conda
 
-if is_pure_conda:
-    from PyInstaller.utils.hooks.conda import collect_dynamic_libs
-    binaries = collect_dynamic_libs('rtree', dest='rtree/lib')
-else:
-    from PyInstaller.utils.hooks import collect_dynamic_libs
-    binaries = collect_dynamic_libs('rtree', destdir='rtree/lib')
+binaries = collect_dynamic_libs('rtree', destdir='rtree/lib')
+if not binaries and is_pure_conda:
```

Comment:
Is there a reason for using `is_pure_conda` instead of just `is_conda`?

Reply:
That should be `is_pure_conda`. The `conda_support` module doesn't exist if the user is running venv inside a conda environment.

Reply:
I thought probably this
https://github.com/pyinstaller/pyinstaller/blob/develop/PyInstaller/compat.py#L165

Reply:
Yeah, `rtree` can't be a conda distribution unless `is_pure_conda` is true.